### PR TITLE
95 [refactor] delete all "any" from ts files and properly type project

### PR DESCRIPTION
**Pull request by 2ujin**
https://github.com/khomch/collabor8/issues/95

### CODE REVIEW

File: client/collabor8/src/app/profile-edit/page.tsx [hunk 1]
```diff
@@ -98,171 +96,172 @@ function ProfileEdit(data: TUserInfo) {
...
-                    name="emailAddress"
-                    label="Email"
-                    placeholder="emailAddress"
-                    status="default"
-                    value={profile?.emailAddress}
-                    onChange={handleChange}
-                  />
-                </div>
+            <div className="profile__input">
+              <div className="prifile__item right">
```
Comment (Kamil-Zmuda) [inline client/collabor8/src/app/profile-edit/page.tsx:120]: Typo in div className

File: client/collabor8/src/app/profile-edit/page.tsx [hunk 2]
```diff
@@ -98,171 +96,172 @@ function ProfileEdit(data: TUserInfo) {
...
+                  type="text"
+                  name="userName"
+                  label="Username"
+                  placeholder="userName"
+                  status="default"
+                  value={profile?.userName}
+                  onChange={handleChange}
+                />
               </div>
+              <div className="prifile__item">
```
Comment (Kamil-Zmuda) [inline client/collabor8/src/app/profile-edit/page.tsx:131]: Typo

File: client/collabor8/src/app/profile-edit/page.tsx [hunk 3]
```diff
@@ -98,171 +96,172 @@ function ProfileEdit(data: TUserInfo) {
...
-                    name="lastName"
-                    label="Lastname"
-                    placeholder="lastName"
-                    status="default"
-                    value={profile?.lastName}
-                    onChange={handleChange}
-                  />
-                </div>
+            <div className="profile__input">
+              <div className="prifile__item right">
```
Comment (Kamil-Zmuda) [inline client/collabor8/src/app/profile-edit/page.tsx:145]: Typo

File: client/collabor8/src/app/profile-edit/page.tsx [hunk 4]
```diff
@@ -98,171 +96,172 @@ function ProfileEdit(data: TUserInfo) {
...
-                    name="company"
-                    label="Company"
-                    placeholder="company"
-                    status="default"
-                    value={profile?.company}
-                    onChange={handleChange}
-                  />
-                </div>
+            <div className="profile__input">
+              <div className="prifile__item right">
```
Comment (Kamil-Zmuda) [inline client/collabor8/src/app/profile-edit/page.tsx:170]: Typo

File: client/collabor8/src/app/profile-edit/page.tsx [hunk 5]
```diff
@@ -98,171 +96,172 @@ function ProfileEdit(data: TUserInfo) {
...
-                    label="Role"
-                    placeholder="role"
-                    status="default"
-                    value={profile?.role}
-                    onChange={handleChange}
-                  />
-                </div>
-                <div className="prifile__item"></div>
+            <div className="profile__input">
+              <div className="prifile__item right">
```
Comment (Kamil-Zmuda) [inline client/collabor8/src/app/profile-edit/page.tsx:197]: Typo

File: client/collabor8/src/app/profile-edit/page.tsx [hunk 6]
```diff
@@ -98,171 +96,172 @@ function ProfileEdit(data: TUserInfo) {
...
-                    name="bio"
-                    label="Bio"
-                    placeholder="bio"
-                    status="default"
-                    value={profile?.bio}
-                    onChange={handleChange}
-                  />
-                </div>
+            <div className="profile__input">
+              <div className="prifile__item right">
```
Comment (Kamil-Zmuda) [inline client/collabor8/src/app/profile-edit/page.tsx:212]: Typo

File: client/collabor8/src/app/profile-edit/page.tsx [hunk 7]
```diff
@@ -98,171 +96,172 @@ function ProfileEdit(data: TUserInfo) {
...
-                    name="tech"
-                    label="Tech Stack"
-                    placeholder="Write Your Tech Stack!"
-                    status="default"
-                    value={techInput}
-                    onChange={(e) => setTechInput(e.target.value)}
-                    onKeyDown={handleKeyDown}
-                  />
+            <div className="profile__input">
+              <div className="prifile__item right">
```
Comment (Kamil-Zmuda) [inline client/collabor8/src/app/profile-edit/page.tsx:226]: Typo

File: client/collabor8/src/app/projects/page.tsx [hunk 8]
```diff
@@ -39,77 +39,41 @@ export default function MyProjects() {
           <div className="projects-page__filters">
             <ProfileCard {...user} />
           </div>
+
           <div className="projects-page__projects">
             {ownerProjects.length > 0 && (
```
Comment (Kamil-Zmuda) [inline client/collabor8/src/app/projects/page.tsx:44]: HTML is broken on this page, it doesn't show headers correctly for:

1. "Owner Projects"
2. "Team member in"
3. "Finished tasks in"
4. Should show: "Not participating in any projects" if user has no projects

**Review by Kamil-Zmuda — COMMENTED** [review]
I commented some issues in your PR.

File: client/collabor8/src/app/projects/page.tsx (hunk 8 above)
Comment (2ujin) [reply client/collabor8/src/app/projects/page.tsx]: I fixed it and pushed it again. :)

**Review by Kamil-Zmuda — APPROVED** [review]
All good now 😄

**Pull request merged**